src/Mono.Android/Mono.Android.csproj

```diff
@@ -361,8 +361,8 @@
     <Compile Include="System.IO\AndroidExtensions.cs" />
     <Compile Include="System.Linq\Extensions.cs" />
     <Compile Condition=" '$(TargetFramework)' != 'monoandroid10' " Include="Xamarin.Android.Net\AndroidClientHandler.cs" />
+    <Compile Condition=" '$(TargetFramework)' != 'monoandroid10' " Include="Xamarin.Android.Net\AndroidMessageHandler.cs" />
```

Comment:
Is it OK to do this now? It broke the apicompat check:
```
/Users/builder/azdo/_work/1/s/xamarin-android/src/Mono.Android/Mono.Android.targets(259,5): error : TypesMustExist : Type 'Xamarin.Android.Net.AndroidMessageHandler' does not exist in the implementation but it does exist in the contract. 
```

Reply:
Can `AndroidMessageHandler` *work* on Classic Xamarin.Android?  If it can, keeping it around would be useful for "migration support".  Or, how hard would it be to provide a useful Classic implementation?

Reply:
I mean you could probably do `new HttpClient(new AndroidMessageHandler())` in classic Xamarin.Android, but we don't have tests in place to know if this even works. I think we just accidentally added the type.

I would think for migration is most people will have `new HttpClient()` and expect the right native handler to be used depending on their project settings?

Reply:
I moved the tests around in https://github.com/xamarin/xamarin-android/pull/6445/commits/3c068980f6a1340d7a2531714f15ab45eca4c48d, and `AndroidMessageHandler` is working. Let's just add it back and see how it goes.